excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/parser/EventCaseFieldParserTest.java

```diff
@@ -67,10 +67,11 @@ public void defDataItemHasValidShowConditionParseEveCaseFldCalledParsedEventEnti
         String originalShowCondition = "Original Show Condition";
         String label = "label";
         String hint = "hint";
+        String defaultValue = "";
```

Comment:
Similarly here, can we add a non-empty value please

Reply:
Fixed in [5e6bcf3](https://github.com/hmcts/ccd-definition-store-api/pull/1111/commits/5e6bcf3530b16e5bff37a8f0c02855a0d686d834)